packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -473,6 +474,7 @@ export class GumbyController {
                 undefined,
                 GumbyNamedMessages.JOB_FAILED_IN_PRE_BUILD
             )
+            await openBuildLogFile()
```

Comment:
Added this so that the build logs open up automatically once downloaded; the `View Logs` button can be difficult to see.